fastlane/lanes/release.rb

```diff
@@ -381,17 +381,19 @@
     end
   end
 
-  # Finalizes a release at the end of a sprint to submit to the App Store
+  # Finalizes a release at the end of a sprint to submit to the App Store, triggering the final release build on CI
   #
-  #  - Updates store metadata
-  #  - Bumps final version number
-  #  - Removes branch protection and close milestone
-  #  - Triggers the final release on CI
+  # This lane performs the following actions:
+  # - Updates store metadata
+  # - Bumps final version number
+  # - Removes branch protection and closes milestone
+  # - Triggers the final release on CI
   #
-  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
+  # @param skip_confirm [Boolean] Whether to skip confirmation prompts
   #
-  desc 'Trigger the final release build on CI'
-  lane :finalize_release do |options, skip_confirm: false|
+  lane :finalize_release do |skip_confirm: false|
+    UI.user_error!('To finalize a hotfix, please use the `finalize_hotfix_release` lane instead') if release_is_hotfix?
```

Comment:
There's something about "`release_is_hotfix?`" that sounds odd.

I was going to suggest adding branch in the name, but that would be incorrect because the check is on the version file not the branch name. 

Maybe `current_version_is_hotfix?` or `is_hotfix?`.

Not sure...

Reply:
I got this method from Woo, which seems to have its name based on a `release-toolkit` method, but I agree -- renamed to `is_current_release_hotfix?` on 51531311b94a2f1dc631ed793f34a878423b96a5.